src/lib.rs

```diff
@@ -48,9 +50,12 @@ pub async fn sync_ldap_users_to_zitadel(config: Config) -> Result<()> {
 
 	sync_handle.await??;
 
-	zitadel.import_new_users(added).await?;
 	zitadel.update_users(changed).await?;
```

Comment:
Won't this still mean that a disabled user being enabled will also propagate from LDAP to Zitadel?

Reply:
No, inside zitadel.rs we have a differentiation between the changes to enable and disable (line 91). I have another check for this feature flag on the zitadel.rs to handle this case. However, I just extended my test case to also cover this scenario